CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+## Getting started
+
+Install dependencies:
+
+```
+bundle install
+```
+
+Compile the gem, run the tests & Ruby linter:
+
+```
+bundle exec rake
+```
+
+## Releasing
+
+1. Bump the `VERSION` in `lib/wasmtime/version.rb`
+1. Create a new tag for that release, prefixed with `v` (`git tag v1.0.0`):
+  
+  ```
+  git tag v$(grep VERSION lib/wasmtime/version.rb | head -n 1 | cut -d'"' -f2)
+  git push --tags
```

Comment:
Not for this PR, I wonder is there's any value in having a small script to automate this.

Reply:
Honestly if feels so small that I don't think it's worth it _yet_. But 👀 [rb-sys has a nice bump release script](https://github.com/oxidize-rb/rb-sys/blob/main/rakelib/release.rake#L2-L17), maybe we could do something similar eventually:
1. ask for the new version
1. update refs in the code
1. commit changes
1. create the tag
1. print "if that all looks good to you, `git push --tags`"